client/src/App/components/NewWorld.tsx

```diff
@@ -1,9 +1,47 @@
-import React from 'react';
+import axios from 'axios';
+// import React, { FormEventHandler, useState, useRef } from 'react';
+import React, { FormEventHandler, useState } from 'react';
 import { Link } from 'react-router-dom';
 
 import { LogoutLink } from './LogoutLink';
 
+export enum RequestState {
+  NOT_STARTED = 'not-started',
+  PENDING = 'pending',
+  SUCCESS = 'success',
+  FAIL = 'fail'
+}
+
 export const NewWorld = () => {
+  const [requestState, setRequestState] = useState<RequestState>(RequestState.NOT_STARTED);
+  // const formRef = useRef<HTMLFormElement>(null);
+
+  const handleSubmit: FormEventHandler<HTMLFormElement> = (submitEvent) => {
+    submitEvent.preventDefault();
+    const form = submitEvent.target as HTMLFormElement;
+    const formData = new FormData(form);
+    // const formData = new FormData(formRef.current || undefined);
+    setRequestState(RequestState.PENDING);
+    formData.append('isNew', 'true');
+    // console.log(formData.get('a'));
+    // console.log(`form data: ${JSON.stringify(formData, null, 2)}`);
+
+    // axios.post('/api/v1/launch', { data: formData })
+    axios.post('/api/v1/launch', {
+      data: formData,
```

Comment:
- [ ] cannot for the life of me get this request body data to show up populated on the server side